src/test/kotlin/org/jitsi/jibri/api/xmpp/XmppApiTest.kt

```diff
@@ -150,6 +151,19 @@ class XmppApiTest : ShouldSpec() {
                         }
                     }
                 }
+                "and jibri is busy" {
+                    whenever(jibriManager.startFileRecording(any(), any(), any(), any())).thenAnswer {
```

Comment:
how does `whenever` `thenAnswer` work? does it mock the method to always throw?

Reply:
Yeah